src/renderer/components/ProjectCreation/ModalContainer.tsx

```diff
@@ -57,11 +62,26 @@ const ModalContainer = ({ isOpen, closeModal }: Props) => {
     }
   };
 
-  const viewComponents = [
-    NewProjectView,
-    ImportMediaView,
-    RunTranscriptionView,
-  ];
+  const viewComponents: any = useMemo(() => {
```

Comment:
Why does this have to be `any`? There should be a type for this

Reply:
Ahh, it was an issue when I had the previous approach implemented. I fixed it here: https://github.com/chloebrett/mlvet/pull/214/commits/1808dd506eb79470d46146ef473f260f6509e7d5